### PR TITLE
docs: recommend redis when users want to use a cache for renovate

### PR DESCRIPTION
**Pull request by tibeer**
The disk cache is very likely to pile up quickly. This will
prevent the Job to be executed once the PersistentVolume is full.
More information can be found here:
https://github.com/renovatebot/renovate/discussions/30525

Signed-off-by: Tim Beermann <tibeer@outlook.de>

### CODE REVIEW

File: charts/renovate/values.yaml [hunk 1]
```diff
@@ -108,6 +108,8 @@ renovate:
   # -- Renovate Container-level security-context
   securityContext: {}
 
+  # Instead of a file system cache, it is highly recommended to use the redis subchart.
+  # Take a look at https://github.com/renovatebot/renovate/discussions/30525 for more information.
```
Comment (viceice) [inline charts/renovate/values.yaml:112]: This should also be in the generated readme. You should also mention that using SQLite cache is also a good alternative

**Review by viceice — CHANGES_REQUESTED** [review]

File: charts/renovate/values.yaml (hunk 1 above)
Comment (tibeer) [reply charts/renovate/values.yaml]: @viceice Thanks for the feedback. I pushed the changes, tell me if this is what was desired :)

**Review by viceice — APPROVED** [review]

File: charts/renovate/values.yaml (hunk 1 above)
Comment (viceice) [reply charts/renovate/values.yaml]: please use the re-request review button next time instead of comment

**Comment by tibeer** [comment]
Something got completly messed up during interactive rebase. I created a new, clean pull request: https://github.com/renovatebot/helm-charts/pull/2099

**Comment by viceice** [comment]
you could simply force push to fix

**Pull request closed without merging**